### PR TITLE
Add 79to88 and 79to89 aws upgrade paths + 79to810 and 810to94

### PR DESCRIPTION
**Pull request by fernflower**
Thanks to the detailed downstream review by mmoran I have realised that upstream upgrade paths have to be revised and updated as well.

### CODE REVIEW

**Comment by fernflower** [comment]
I guess this one will also be blocked on infra issues :(

File: .packit.yaml [hunk 1]
```diff
@@ -487,7 +505,7 @@ jobs:
   targets:
     epel-8-x86_64:
       distros: [RHEL-8.6-rhui]
-  identifier: sanity-8to9-aws-e2e
+  identifier: sanity-8.6to9.0-aws-e2e
```
Comment (pirat89) [inline .packit.yaml:508]: @fernflower  @mm0ran  I guess it should be set 8.8to9.2 instead, as 8.6 to 9.0 is going to be dropped (I already wanted to drop it) from the enabled upgrade paths and we do not consider any other development related to this upgrade path, hence no testing.

The same about the `sanity-79to86-aws`, use rather 79to88. Thinking about that, 8.9 and 9.3 are outside. We want to drop these also, but I can keep them till Feb if wanted.

Comment (pirat89) [reply .packit.yaml:508]: Update: we are for now keeping also 8.6 & 9.0 upgrade paths until tests are updated.

**Review by pirat89 — APPROVED** [review]

**Pull request merged**